pages/docs/couple-your-code/couple-your-code-time-step-sizes.md

```diff
@@ -75,6 +75,45 @@ This procedure is independent of whether a serial or a parallel coupling scheme
 For parallel coupling, both solvers run together and everything happens simultaneously in both participants, while for serial coupling, the first participant needs reach the end of the window before the second one can start.
 {% endnote %}
 
+### Possible subcycling pitfall
```

Comment:
I will remove this section from this PR. It's not really the same topic and it's I distracting from the main issue. We can deal with this elsewhere.